ui/app/templates/partials/transit-form-edit.hbs

```diff
@@ -91,14 +91,12 @@
         </div>
       </div>
       {{#if (and key.canDelete capabilities.canDelete)}}
-        {{#confirm-action
-          buttonClasses="button"
-          onConfirmAction=(action "deleteKey")
-          confirmMessage=(concat "Are you sure you want to delete " key.id "?")
-          cancelButtonText="Cancel"
-        }}
+        <ConfirmAction
+          @buttonClasses="button"
+          @onConfirmAction={{action "deleteKey"}}
+        >
           Delete encryption key
```

Comment:
I don't think I noticed we call this an encryption key - This should probably just be `Delete key` or `Delete transit key`

Reply:
Yeah, I usually call it "transit key". I like that better :+1: